tests/unit/test_tenant.py

```diff
@@ -10,6 +12,28 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
+    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
+        tenant = create_tenant()
+
+        # Create an expired lease for the tenant
+        lease_expired_end = faker.date_time_this_decade(
+            before_now=True, after_now=False
+        )
+        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
+        lease_expired = create_lease(
+            tenant=tenant,
+            dateTimeStart=lease_expired_start,
+            dateTimeEnd=lease_expired_end,
+        )
+
+        # Create an active lease for the tenant
+        lease_active = create_lease(
+            tenant=tenant, property=PropertyModel.find(lease_expired.propertyID)
+        )
+
+        # Active lease should be the only one that shows up
+        assert tenant.json()["lease"] == lease_active.json()
```

Comment:
This is testing the json method, but I would have expected it to test the active method.

Reply:
I would expect a separate json test to test the entirety of the json method.

Reply:
so when we test the active method here. We can test the happy and sad path. When we test the json method then we can test the happy path.